src/views/Notes/NoteCreatePage.test.js

```diff
@@ -0,0 +1,28 @@
+import { createMemoryHistory } from 'history';
+
+import renderWithRouter from 'helpers/renderWithRouter';
+
+import NoteCreatePage from './NoteCreatePage';
+
+const history = createMemoryHistory();
+history.goBack = jest.fn();
+
+const props = {
+  history,
+  location : history.location,
+};
+
+const renderNoteCreatePage = () => renderWithRouter(
+  <div>
+    <NoteCreatePage {...props} />
+    Note Create Page
+  </div>
+);
+
+describe('Note Create Page', () => {
+  it('should render Note Create Page', async () => {
+    const { getByText } = renderNoteCreatePage();
+
+    expect(getByText('Note Create Page')).toBeInTheDocument();
```

Comment:
@OybekMamatovEPAMSystems it looks like we are testing something which is provided in `renderNoteCreatePage ` above but not really anything related to `<NoteCreatePage />`. I don't think this is needed and we should instead try to test `NoteCreatePage`.